src/api.js

```diff
@@ -133,7 +141,11 @@ const Api = (options = {}) => {
     if (params.filter(p => p.position === 'path').some(p => request.indexOf(`{${p.nameOriginal}}`) === -1)) {
       throw new Error('Path Parameter not defined in given path.');
     }
+    if (params.filter(p => p.isFieldsParam).length > 1) {
```

Comment:
Change this so is only one fields parameter that has auto prune instead of one fields parameter